### ISSUE TITLE
[Feature] Get contents of deleted file

### ISSUE THREAD

**Issue by SamuelMolling** [tj-actions/changed-files#938]
### Is this feature missing in the latest version?

- [X] I'm using the latest release

### Is your feature request related to a problem? Please describe.

Basically I'm using this action to get all the files that were changed/created and run terragrun on top of each one. My problem today is that for me to destroy a resource in terragrunt, I need the file. What I would like is to take the file that was deleted, keep it until the end of the workflow execution and delete it, so that the resources can be deleted.

### Describe the solution you'd like?

It would be something like:

      - name: Run changed-files with dir_names
        id: changed-files-dir-names
        uses: tj-actions/changed-files@v35
        with:
          dir_names: "true"
          content_file_tmp: "true"
          files: |
            terraform/mongodb-atlas/clusters/**

      - name: Terragrunt Destroy
        id: destroy
        run: |
          for folder in ${{ steps.changed-files-dir-names.outputs. file_tmp }}; do
            terragrunt destroy -auto-approve --terragrunt-working-dir $folder
          done 

### Describe alternatives you've considered?

I couldn't think of many things at the moment.

In summary, I would like to run my destroy on the file before it is deleted or get the state prior to deletion, mount it and run. then discard it.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jackton1** [comment]
@SamuelMolling does the file need to still be located in the working tree or can it reside in a temp directory?

**Comment by SamuelMolling** [comment]
The file needs to reside in the directory, on account of the state file, located in a bucket in the google cloud

**Comment by SamuelMolling** [comment]
Also, I would like to know if it is possible to run on all modified files excluding one.

**Comment by SamuelMolling** [comment]
To solve the problem, I made an if, like this.

          for folder in ${{ steps.changed-files-dir-names.outputs.all_changed_files }}; do
            if [ -f $folder/cluster.hcl ]
            then
              echo "cluster.hcl found"
            else
              terragrunt init --terragrunt-working-dir $folder
            fi
          done

**Comment by jackton1** [comment]
Given the complexity of reverting the state of the repository in order support this feature, I’ll recommend your workflow destroys the resource via inputs if necessary and deletes the related folder prior to committing the changes.

**Comment by SamuelMolling** [comment]
@jackton1 Do you have an idea how to do it? I'm not sure how I would do it. Would you have to open 2 prs there? One to destroy and another to delete the files, right?

**Comment by SamuelMolling** [comment]
@jackton1 It is not possible to recover the value of the deleted file in PR and create it temporarily in the container and then remove it, I believe it is not so complicated to get this value, there is a load diff, basically it would be to get the value before it for deleted files .

**Comment by SamuelMolling** [comment]
Se tiver como recuperar o valor do arquivo deletado, eu consigo fazer a lógica localmente para criação e depois deletar o mesmo.

**Comment by SamuelMolling** [comment]
Could this help? https://github.com/marketplace/actions/get-diff-action
